Makefile

```diff
@@ -173,7 +173,7 @@ $(webhook_cert): $(webhook_key)
 
 
 .PHONY: webhook-debug
-webhook_service_name = host.docker.internal
+webhook_service_name = host.docker.internal ## For Linux users: instead of rewriting below logic to adapt to your setup, just add a line to your /etc/hosts file, for example: 172.17.0.1 host.docker.internal
```

Comment:
you can just do `make webhook-debug -e webhook_service_name= 172.17.0.1`

Reply:
won't work, You need some name otherwise it breaks when using just IPs

Reply:
really? i specifically tested it with IPs as well.

Did you delete the old certificates beforehand? They are not regenerated if you change the hostname.

Reply:
deleting certificates directory solved the issue, I'll cleanup this comment